go.mod

```diff
@@ -3,45 +3,54 @@ module github.com/moshloop/platform-cli
 go 1.12
 
 require (
+	github.com/alecthomas/kingpin v2.2.6+incompatible
 	github.com/aws/aws-sdk-go v1.29.5
-	github.com/davecgh/go-spew v1.1.1
```

Comment:
Go-spew is used, and crunchydata isn't ?

Reply:
It should be a mistake. Will change.